### PR TITLE
widen the analyzer package constraints

### PR DESCRIPTION
**Pull request by keertip**
@jakemac53

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+environment:
+  sdk: '>=1.0.0 <2.0.0'
 dependencies:
-  analyzer: ">=0.15.6 <0.26.0"
-  barback: ">=0.14.2 <0.16.0"
-  cli_util: ">=0.0.1 <0.1.0"
-  path: ">=0.9.0 <2.0.0"
-  source_maps: ">=0.9.4 <0.11.0"
-  source_span: ">=1.0.0 <2.0.0"
+  analyzer: '>=0.25.0 <0.27.0'
```
Comment (jakemac53) [inline pubspec.yaml]: can you actually leave the lower bound at 0.15.6 for this package, unless there was an issue that came up?

**Comment by keertip** [comment]
@jakemac53  fixed the tests, PTAL.

**Comment by jakemac53** [comment]
cc @sigmundch

**Comment by jakemac53** [comment]
thanks!

**Pull request merged**